utilities/similarity/entrypoint.py

```diff
@@ -0,0 +1,203 @@
+        parser_train.add_argument('-mode', required=False, default='alignment',
+                            help='mode (alignment, sentence) [alignment]')
+        parser_train.add_argument('-max_sents', required=False, type=int, default=0,
+                            help='Consider this number of sentences per batch (0 for all) [0]')
+        parser_train.add_argument('-n_epochs', required=False, type=int, default=1,
+                            help='train for this number of epochs [1]')
+        parser_train.add_argument('-report_every', required=False, type=int, default=1000,
+                            help='report every this many batches [1000]')
+
+        parser_apply = subparsers_similarity.add_parser('simapply', parents=[parser_similarity],
+                            add_help=False,
+                            help='similarity module apply mode')
+        parser_apply.add_argument('-tst_src', required=True, help='testing data, source')
+        parser_apply.add_argument('-tst_tgt', required=True, help='testing data, target')
+        parser_apply.add_argument('-epoch', required=False, type=int,
+                            help='epoch to use ([mdir]/epoch[epoch] must exist, by default the latest one in mdir)')
+        parser_apply.add_argument('-output', required=False, default='-',
+                            help='output file [- by default is STDOUT]')
+        parser_apply.add_argument('-q', required=False, action='store_true',
+                            help='quiet mode, just output similarity score')
+        parser_apply.add_argument('-show_matrix', required=False, action='store_true',
+                            help='output formatted alignment matrix (mode must be alignment)')
+        parser_apply.add_argument('-show_svg', required=False, action='store_true',
+                            help='output alignment matrix using svg-like html format (mode must be alignment)')
+        parser_apply.add_argument('-show_align', required=False, action='store_true',
+                            help='output source/target alignment matrix (mode must be alignment)')
+        parser_apply.add_argument('-show_last', required=False, action='store_true',
+                            help='output source/target last vectors')
+        parser_apply.add_argument('-show_aggr', required=False, action='store_true',
+                            help='output source/target aggr vectors')
+
+    def exec_function(self, args):
+        new_args = []
+        local_output = None
+
+        new_args.append('-mdir')
+        local_model_dir = self.convert_to_local_file([args.mdir], is_dir=True)[0]
+        new_args.append(local_model_dir)
+        new_args.append('-batch_size')
+        new_args.append(str(args.batch_size))
+        new_args.append('-seed')
+        new_args.append(str(args.seed))
+        if args.debug:
+            new_args.append('-debug')
+        if args.cmd == 'simtrain':
+            new_args.append('-trn')
```

Comment:
This could be made a bit more compact using:

```python
new_args.extend([
    '-trn', self.convert_to_local_file([args.trn])[0],
    '-dev', self.convert_to_local_file([args.dev])[0],
    ....
])
```